base/some.jl

```diff
@@ -6,6 +6,8 @@
 A wrapper type used in `Union{Some{T}, Nothing}` to distinguish between the absence
 of a value ([`nothing`](@ref)) and the presence of a `nothing` value (i.e. `Some(nothing)`).
 
+`Some` is also used in broadcasting to treat it's enclosed value as a scalar.
```

Comment:
Maybe more accurate to say

```suggestion
`Some` is also used in broadcasting to broadcast a single value to all axes.
```

? Though I see that a similar wording is already used in `Ref`.

Reply:
I feel like that suggestion is less likely to be understood. However, maybe not everyone knows the word "scalar" either.

Reply:
Maybe it's fine as-is if we also explain the actual interface it implements (i.e., it is a 0-dim iterable that supports indexing).

Reply:
This should say `its` rather than `it's` in the original formulation.

Reply:
Thanks Yuri!